clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -937,13 +936,341 @@ check_schedule_file()
     return true;
 }
 
+bool
+check_branch_decoration()
+{
+    std::cerr << "Testing branch decoration\n";
+    // Indirect branch target: correct.
```

Comment:
Thanks for changing the "positive"/"negative" with "correct"/"incorrect". The former was confusing.

Reply:
Agreed.